src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/ActionSvcClientImplTest.java

```diff
@@ -124,4 +129,106 @@ public void testCreateActionPlanRestClientException() {
 
     // Then RestClientException is thrown
   }
+
+  /** Test that the action service is called with the correct details when creating action rules. */
+  @Test
+  public void testCreateActionRule() {
+    // Given
+    ActionSvc actionSvcConfig = new ActionSvc();
+    actionSvcConfig.setActionRulesPath(ACTION_RULE_PATH);
+    when(appConfig.getActionSvc()).thenReturn(actionSvcConfig);
+
+    UriComponents uriComponents =
+        UriComponentsBuilder.newInstance()
+            .scheme(HTTP)
+            .host(LOCALHOST)
+            .port(80)
+            .path(ACTION_PATH)
+            .build();
+    when(restUtility.createUriComponents(any(String.class), any(MultiValueMap.class)))
+        .thenReturn(uriComponents);
+
+    ActionRulePostRequestDTO actionRulePostRequestDTO = new ActionRulePostRequestDTO();
```

Comment:
Might be worth refactoring to some smaller methods to make the test easier to understand and smaller :)